### PR TITLE
feat: add option to pretty print hub measurements to file

### PR DESCRIPTION
**Pull request by troian**
Add config option to pretty print hub measurements to file
Useful for debug purpose

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -38,6 +38,9 @@ type MinValuableConfig struct {
 	HubPassword string   `toml:"hub_password" commented:"true"`
 }
 
+type ConfigsLogs struct {
+	HubFile string `toml:"hub_file,omitempty" comment:"log hub objects send to the hub"`
+}
```
Comment (nikita-vanyasin) [inline config.go]: Empty line missing

File: config.go [hunk 2]
```diff
@@ -38,6 +38,9 @@ type MinValuableConfig struct {
 	HubPassword string   `toml:"hub_password" commented:"true"`
 }
 
+type ConfigsLogs struct {
```
Comment (nikita-vanyasin) [inline config.go]: Not sure why both words in the type name are plural. I think it's better to name like `LogFilesConfig`

File: config.go (hunk 1 above)
Comment (troian) [reply config.go]: done

File: config.go (hunk 2 above)
Comment (troian) [reply config.go]: done

**Pull request merged**